e2e/tests/functional/search.e2e.spec.js

```diff
@@ -139,68 +137,86 @@ test.describe("Search Tests @unstable", () => {
         await expect(searchResults).toHaveText(folderName);
     });
 
+    test('Search results are debounced @couchdb', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/6179'
+        });
+        await createObjectsForSearch(page);
+
+        let networkRequests = [];
+        page.on('request', (request) => {
+            const searchRequest = request.url().endsWith('_find');
+            const fetchRequest = request.resourceType() === 'fetch';
+            if (searchRequest && fetchRequest) {
+                networkRequests.push(request);
+            }
+        });
+
+        // Full search for object
+        await page.type("input[type=search]", 'Clock', { delay: 100 });
+
+        // Wait for search to finish
+        await waitForSearchCompletion(page);
+
+        // Network requests for the composite telemetry with multiple items should be:
+        // 1.  batched request for latest telemetry using the bulk API
+        expect(networkRequests.length).toBe(1);
+
+        const searchResultDropDown = await page.locator(searchResultDropDownSelector);
+
+        await expect(searchResultDropDown).toHaveText('Clock A');
+    });
+
     test("Validate multiple objects in search results return partial matches", async ({ page }) => {
         test.info().annotations.push({
             type: 'issue',
             description: 'https://github.com/nasa/openmct/issues/4667'
         });
 
-        // Go to baseURL
-        await page.goto("/", { waitUntil: "networkidle" });
-
         // Create folder objects
-        const folderName = "e928a26e-e924-4ea0";
+        const folderName1 = "e928a26e-e924-4ea0";
         const folderName2 = "e928a26e-e924-4001";
 
-        await createFolderObject(page, folderName);
-        await createFolderObject(page, folderName2);
+        await createDomainObjectWithDefaults(page, {
+            type: 'Folder',
+            name: folderName1
```

Comment:
We can remove the `name` option altogether, and use the returned objects `.name` property instead (it will generate a unique name if no name is provided).

Reply:
I think the purpose of the test though is to validate multiple matches on a single substring. If we generated two unique names, we won't be able to find both of them (easily) with a single substring.

Reply:
Ah, that's right. This is good as is then!